create.js

```diff
@@ -0,0 +1,32 @@
+import WBEdit from 'wikibase-edit'
+import {data} from "./Data Model/mainProperties.js";
```

Comment:
Instead of using data stored in `.js` files:
```
import {data} from "./Data Model/mainProperties.js";
```
it would be better to use more wikidatabase-native formats, e.g. [JSON](https://doc.wikimedia.org/Wikibase/master/php/docs_topics_json.html) - [recommended dump format for Wikidata](https://www.wikidata.org/wiki/Wikidata:Database_download#JSON_dumps_(recommended)).

It would also be advisable to use ready-made tools prepared for wikibase editing, e.g.: [Wikibase-CLI](https://github.com/maxlath/wikibase-cli).

Reply:
It is not such obvious that data stored as an js object is a wrong decision. There are some advantages of that decision. For example objects allows to keep comments inside instead of JSON. Even a framework which you pointed allows implementation using JS objects :) 
https://github.com/maxlath/wikibase-cli/blob/main/docs/write_operations.md#pass-data-as-a-static-js-object-file 

My suggestion is to keep model as js objects, further data could be in jsons. 
I can remove my wiki-edit implementation and change a Readme file to describe how use a tool what you suggested.

Reply:
The main problem is the incompatibility of the format used in `mainProperties.js` file with the recommended existing standards.

The most flexible solution will most likely be the method described in [Generate an edit object from a JS template function](https://github.com/maxlath/wikibase-cli/blob/main/docs/write_operations.md#generate-an-edit-object-from-a-js-template-function) and, depending on the specific situation, the use of:

- universal [Wikibase-CLI](https://github.com/maxlath/wikibase-cli) tool , or ...
- .. preparing your own simple code based on [wikibase-sdk](https://github.com/maxlath/wikibase-sdk)

---
Try to execute the following command:
```bash
wd gt -i https://wiki.kul.pl/w/api.php Q1
```
which returns:
```js
module.exports = function () {
  return {
    id: 'Q1',
    type: 'item',
    labels: {
      pl: 'Osoba',
      de: 'Person',
      en: 'Person',
      fr: 'Personne',
      pt: 'Pessoa',
      el: 'Πρόσωπο',
      ru: 'Личность',
      zh: '人物'
    },
    descriptions: {},
    aliases: {},
    claims: {},
    sitelinks: {}
  }
}
```

and for comparision:
```bash
wd gt -i https://wiki.kul.pl/w/api.php -j Q1
```
which returns:
> {"id":"Q1","type":"item","labels":{"pl":"Osoba","de":"Person","en":"Person","fr":"Personne","pt":"Pessoa","el":"Πρόσωπο","ru":"Личность","zh":"人物"},"descriptions":{},"aliases":{},"claims":{},"sitelinks":{}}